docs/docs/integrations/providers.md

```diff
@@ -1,3 +1,32 @@
-# Provider Integrations
+# Providers
+
+A provider connects Minder to your software supply chain. It lets Minder know where to look for your repositories, artifacts,
+and other entities are, in order to make them available for registration. It also tells Minder how to interact with your
+supply chain to enable features such as alerting and remediation. Finally, it handles the way Minder authenticates
+to the external service.
+
+The currently supported providers are:
+* GitHub
+
+Stay tuned as we add more providers in the future!
+
+## Enrolling a provider
+
+To enroll GitHub as a provider, use the following command:
+```
+minder provider enroll
+```
+
+To enroll a specific GitHub organization, use the following command:
+```
+minder provider enroll --owner specific-org
```

Comment:
The `--owner` flag only applies to the legacy `github` provider, since the GitHub App allows the selection in the browser. Do you think it should still be part of the docs here?

Reply:
I think it's fine for the upstream docs. I'll have a dedicated PR for downstream